spec/resolver-fixtures/relative-path.yml

```diff
@@ -120,3 +145,10 @@ expectation: |
     role: !role
       account: the-account
       id: myapp/the-group
+  - !revoke
+    member: !member
+      account: the-account
+      id: myapp/../the-group
```

Comment:
Did you expect this `..` to be resolved?

Reply:
I did not, I forgot to change this from an earlier test I was running.